ci/Jenkinsfile.android

```diff
@@ -64,7 +64,11 @@ pipeline {
     }
     stage('Build') {
       steps {
-        script { cmn.nix.shell('lein prod-build-android')}
+        script {
+          cmn.nix.shell('cp params_prod.edn params.edn')
+          cmn.nix.shell('lein prod-build-android')
+          cmn.nix.shell('node prepare-modules.js')
```

Comment:
Wouldn't it make more sense to just do `make prod-build-android` instead of repeating exactly the same thing you have in the Makefile?

Reply:
will rewrite later today. That's not my focus usually, I'm satisfied if this part works at all :)

Reply:
#8360 was merged and does this already by using the Makefile target.